app/models/medium.rb

```diff
@@ -102,6 +99,17 @@ def put_file_with_retry(file_path, opts={}, num_retries=5)
     raise e
   end
 
+  def validate_content_type
+    errors.add(:content_type, 'Invalid file type') if BLOCKED_CONTENT_TYPES.match?(content_type)
+
+    export_medium = type.match?(EXPORT_MEDIUM_TYPE_REGEX)
+    validate_export_medium if export_medium
```

Comment:
Not blocking but i don't really like how this is lumped in here though i can see it's logically validating the content type.

I think extracting this to it's own custom validation makes more sense
```
validate :validate_content_type
validate :validate_media_content_type if: :export_medium
```

Reply:
ah i must have misunderstood what you meant initially on https://github.com/zooniverse/panoptes/pull/4026#discussion_r1029982988.  i'm ok with updating.